azkaban-web-server/src/main/java/azkaban/imagemgmt/models/BaseModel.java

```diff
@@ -0,0 +1,39 @@
+package azkaban.imagemgmt.models;
+
+public abstract class BaseModel {
+  private String createdBy;
```

Comment:
id can also come here. Can you please check if type of createdOn and ModifiedOn can be converted to date and time?

Reply:
Yes added id in the BaseModel. Regarding createdOn and modifiedOn replied in the previous comment.